backend/app/app.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any, Dict, Tuple
+
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def status() -> Tuple[Dict[str, Any], int]:
+    return {"response": "Welcome to Landing Page"}, 200
```

Comment:
naming structure of methods needs a documented reason

Reply:
What ya want me to change the name to? and we also need to setup docs, that's for another time.